lib/plugins/globalid.js

```diff
@@ -4,8 +4,8 @@
 
 // Modules
 var jsonp = require('jsonp')
-var _ = require('../utils/lodash')
-var invariant = require('invariant')
+var assert = require('../utils/misc').assert
+var noop = require('../utils/misc').noop
```

Comment:
I figure the difference isn't noticeable. But when I'm importing it from lodash it's exported as `function noop () {} \n module.exports = noop` vs `function noop () {} \n exports.noop = noop`, so I think that's like 2 characters less?